### DISCUSSION TITLE
lag corner case: active-standby mlag without any lag specific config on the switches

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#1640] [Features]
PR https://github.com/ipspace/netlab/pull/1624 contains a special use case involving active-standby lag to a pair of switches.
This use case can work without any lag support on the M side

It is a corner case, and I could imagine we may not want to mix it up with adding Linux lag support

Thoughts / opinions? Too much of a corner case to worry about?

If we do decide to support it: How would we model this? Some kind of ```dummy``` interfaces on the M-side that get removed during cleanup perhaps?

**Comment by ipspace** [comment]
We had that discussion already (#1504). Do you think something has changed in a month?

I told you that active-standby bonding in Linux is not LAG, and I told you to implement it in a plugin. This will not get merged as part of the **lag** module _because it is not LAG_ (and I just realized we might have to adjust the LAG module documentation).

**Reply by jbemmel** [reply]
Then we should probably remove ‘’’active-backup’’’ as a lag mode option value too?

I can make the changes, probably Friday

**Reply by ipspace** [reply]
> Then we should probably remove ‘’’active-backup’’’ as a lag mode option value too?

Is it used anywhere else? If not, let's remove it (although active-backup is a configuration in some LAG implementations).

> I can make the changes, probably Friday

No rush. Add them to the Linux LAG PR whenever that one is ready.
